go.mod

```diff
@@ -6,10 +6,14 @@ require (
 	github.com/golang/snappy v0.0.3
 	github.com/minio/sha256-simd v1.0.0
 	github.com/mitchellh/mapstructure v1.3.2
+	github.com/prysmaticlabs/gohashtree v0.0.1-alpha.0.20220714111606-acbb2962fb48
+	github.com/stretchr/testify v1.8.1
```

Comment:
I added `stretchr` for testing assertions - it makes checking errors and equality a little simpler.